app/decorators/controllers/spree/admin/user_sessions_controller_decorator.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../action_controller_base_decorator.rb'
+
 Spree::Admin::UserSessionsController.class_eval do
```

Comment:
Do you think you could refactor this without `class_eval` ?

Reply:
Not sure
- how you would want to do this? In any case you need to add funtionality to the UserSessionController. That's nothing that I added doing.
- how this would affect the load issue topic, as it would still need to load action_controller_base_decorator.rb first to have access to `:detect_spreefinery_single_sign_on!`